app/views/subs.py

```diff
@@ -298,6 +298,12 @@ def create_sub():
     except Sub.DoesNotExist:
         pass
 
+    if config.site.sub_creation_admin_only == True:
+        if (not current_user.admin):
```

Comment:
This condition could be simplified to `config.site.sub_creation_admin_only and not current_user.admin`